inst/examples/text/app.R

```diff
@@ -6,43 +6,62 @@ app <- shinyApp(
     f7SingleLayout(
       navbar = f7Navbar(title = "Text inputs"),
       f7Block(f7Button("update", "Click me")),
-      f7Block(
+      f7BlockTitle("A list of inputs"),
+      f7List(
```

Comment:
it can actually be `f7Block` since when omitting `f7List` all input items are automatically wrapped within `f7List`.